pages/api/faucet/mumbai.ts

```diff
@@ -0,0 +1,81 @@
+import { ChainId } from "@thirdweb-dev/react";
+import { EVM_RPC_URL_MAP } from "constants/rpc";
+import { ethers } from "ethers";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method !== "POST") {
+    return res.status(400).json({ message: "Please use POST request" });
+  }
+
+  const { address } = JSON.parse(req.body);
+
+  const wallet = new ethers.Wallet(
+    process.env.PRIVATE_KEY as string,
+    new ethers.providers.JsonRpcProvider(EVM_RPC_URL_MAP[ChainId.Mumbai]),
+  );
+
+  const data = JSON.stringify({
+    jsonrpc: "2.0",
+    id: 0,
+    method: "alchemy_getAssetTransfers",
+    params: [
+      {
+        fromBlock: "0x0",
+        fromAddress: "0xdf62A3fb98B160D98b406F055EdE42FB26C7327B",
+        toAddress: address,
+        category: ["external"],
+      },
+    ],
+  });
```

Comment:
this will not work once we switch to our own public RPC (soon) because it's an alchemy specific call (alchemy_ prefix) and it is not guaranteed to support that